src/OrleansServiceFabricUtils/OrleansServiceListener.cs

```diff
@@ -21,7 +21,21 @@ public static class OrleansServiceListener
         /// <returns>A <see cref="ServiceInstanceListener"/> which manages an Orleans silo.</returns>
         public static ServiceInstanceListener CreateStateless(ClusterConfiguration configuration)
         {
-            return new ServiceInstanceListener(context => new OrleansCommunicationListener(context, configuration), OrleansServiceFabricEndpointName);
+            return new ServiceInstanceListener(
+                context => new OrleansCommunicationListener(context, configuration),
+                OrleansServiceFabricEndpointName);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="ServiceInstanceListener"/> which manages an Orleans silo for a stateless service.
+        /// </summary>
+        /// <param name="configuration">The Orleans cluster configuration.</param>
+        /// <returns>A <see cref="ServiceInstanceListener"/> which manages an Orleans silo.</returns>
+        public static ServiceReplicaListener CreateStateful(ClusterConfiguration configuration)
```

Comment:
I do intend to add such a sample, but the intention was to add one when we have deeper integration for stateful services. The code for stateless and stateful services is identical except for the type returned - so I'm comfortable including both without having samples for both.

If you rather, I can include a 'stateful' calculator service now, too - but the stateful part would be a misnomer since the Orleans part couldn't reasonably store state within SF until we specifically support that.